napari/layers/utils/color_manager.py

```diff
@@ -463,12 +463,15 @@ def _from_layer_kwargs(
         ] = None,
         color_mode: Optional[Union[ColorMode, str]] = None,
         current_color: Optional[np.ndarray] = None,
-        default_color_cycle: np.ndarray = np.array([1, 1, 1, 1]),
+        default_color_cycle: np.ndarray = None,
```

Comment:
I think it would be nice to have this actually be a default, instead of `None`. Maybe this?:

```suggestion
        default_color_cycle: ColorType = ColorArray('white'),
```

Reply:
it violates B008

Reply:
I guess we could make a module level default constant, but feel free to leave this out.